develop/package_basics.Rmd

```diff
@@ -0,0 +1,182 @@
+---
+title: "Creating a new R package for population genetics"
+---
+
+Motivation
+=========
+
```

Comment:
It allows me to easily see the difference between main headers and sub-headers, but I can change it to a consistent style.